internal/command/instance_model.go

```diff
@@ -81,6 +83,6 @@ func (wm *InstanceWriteModel) Query() *eventstore.SearchQueryBuilder {
 		Builder()
 }
 
-func InstanceAggregateFromWriteModel(wm *eventstore.WriteModel) *eventstore.Aggregate {
+func InstanceAggregateFromWriteModel(ctx context.Context, wm *eventstore.WriteModel) *eventstore.Aggregate {
```

Comment:
Please change this function back so that there is no context as parameter and have a look at for example ExecutionAggregateFromWriteModel to solve the linter issue.

Reply:
Ok, thanks @stebenz  for the hint, then do you mean I should use c.pushAppendAndReduce to fix the lint issues, right?

Reply:
Absolutely @stebenz

Reply:
The linter issue for InstanceAggregateFromWriteModel came from the ctx which is no parameter right?
So if you change the content of the InstanceAggregateFromWriteModel function to be similar to ExecutionAggregateFromWriteModel without the ctx then a lot of changes in this PR should be reverted.